src/js/me-i18n.js

```diff
@@ -112,34 +112,101 @@
 		},
 
 		/**
-		 * Translate a string to a specified language
+		 * Translate a string to a specified language, including optionally a number to pluralize translation
 		 *
 		 * @param {String} input
+		 * @param {Number} pluralReplacement
```

Comment:
Thanks for pointing this. I'll change the argument names to `message` and `pluralParam`. Originally I had it with arguments, but like you stated it's not really needed for our purposes